app/components/Modals/EditDeparmentModal/EditDepartmentModal.jsx

```diff
@@ -17,10 +17,11 @@ function EditDepartmentModal({ department, onClose }) {
   const [currentUser, setCurrentUser] = useState({});
   const [currentSubstitute, setCurrentSubstitute] = useState({});
   const [showDropdown, setShowDropdown] = useState(false);
+  const [isActive, setIsActive] = useState(department.is_active);
```

Comment:
Don't use your props inside useState, instead use useEffect to update your state.

https://medium.com/@digruby/do-not-use-props-as-default-value-of-react-usestate-directly-818ee192f454

Reply:
Thanks! 
I fixed it.